generate.go

```diff
@@ -9,9 +9,8 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/Sirupsen/logrus"
+	"github.com/opencontainers/ocitools/libgen"
```

Comment:
There was a reasonable amount of “what should we call this?” discussion in #128, which resolved around [`seccomp-gen`][1].  If we're aiming for separate generate/validate/([test][1][?][2]) commands, we probably want something like:

    ocitools
    |-- cmd
    |    `-- ocitools
    |         |-- main.go
    |         |-- generate.go
    |         |-- validate.go
    |         …
    |-- generate
    |    |-- generate.go
    |    |-- utils.go, etc.
    |    `-- seccomp (#128)
    |         |-- consts.go
    |         …
    |-- validate
    |    |-- validate.go
    |    `-- utils.go
    …

Shifting the import paths around while we settle into whatever layout we choose isn't a big deal as long as we don't cut releases.  But I expect that the libraries will start picking up external consumers shortly after they're born, and we'll make life easier on those downstream folks if we don't churn the layout too much ;).

[1]: https://github.com/opencontainers/ocitools/pull/128#issuecomment-230688108
[2]: https://github.com/opencontainers/ocitools/pull/47#issuecomment-215328958
[3]: https://github.com/opencontainers/ocitools/pull/61#issuecomment-217910342